proxy-node-shared/src/main/java/uk/gov/ida/notification/SamlFormViewBuilder.java

```diff
@@ -10,13 +10,13 @@
 public class SamlFormViewBuilder {
     private SamlObjectMarshaller marshaller = new SamlObjectMarshaller();
 
-    public SamlFormView buildRequest(String url, AuthnRequest authnRequest, String submitText, String relayState) {
+    public SamlFormView buildRequest(String url, AuthnRequest authnRequest, String relayState) {
         String samlMessage = marshaller.transformToString(authnRequest);
         String encodedSamlMessage = Base64.encodeAsString(samlMessage);
-        return buildRequest(url, encodedSamlMessage, submitText, relayState);
+        return buildRequest(url, encodedSamlMessage, relayState);
     }
 
-    public SamlFormView buildRequest(String url, String encodedSamlMessage, String submitText, String relayState) {
+    public SamlFormView buildRequest(String url, String encodedSamlMessage, String relayState) {
```

Comment:
It looks like `submitText` wasn't used in this method anyway?

Reply:
Yep - this is just cleaning things up